.buildkite/pipeline.yml

```diff
@@ -62,14 +62,21 @@ steps:
           packages.baseDAO-ligo-meta.exes.baseDAO-ligo-meta
         -c bats ./scripts/test.bats
 
-  - label: test
+  - label: test-local-chain-007
+    env:
+      TASTY_NETTEST_NODE_ENDPOINT: "http://localhost:8735"
     if: *not_scheduled
-    commands:
+    commands: &test
     - nix-build ci.nix -A packages.baseDAO.tests.baseDAO-test
-    # Emulator + local chain for Delphi protocol
+    - export TASTY_NETTEST_DATA_DIR="$(mktemp -d --tmpdir="$PWD")"
     - nix run -f ci.nix tezos-client -c
       ./result/bin/baseDAO-test --nettest-run-network
-      -A localhost -P 8735
+
+  - label: test-local-chain-008
+    env:
+      TASTY_NETTEST_NODE_ENDPOINT: "http://localhost:8734"
+    if: *not_scheduled
+    commands: *test
 
   - label: ligo-test
```

Comment:
Please update ligo tests to work on both chains as well, an open PR should be merged soon to run the tests on Nettest as well.

Also, judging from CI, you also forgot to update `callFrom` in the Haskell module of the LIGO version.

Reply:
Right, I've missed that ligo part is covered by a separate haskell project :confused: